OloEngine/src/OloEngine/Physics3D/JoltJobSystemAdapter.cpp

```
@@ -0,0 +1,85 @@
+    JoltJobSystemAdapter::JoltJobSystemAdapter(u32 inMaxJobs, u32 inMaxBarriers)
+    {
+        // Initialize the barrier system (provided by JobSystemWithBarrier base class)
+        JPH::JobSystemWithBarrier::Init(inMaxBarriers);
+
+        // Initialize the job free list
+        m_Jobs.Init(inMaxJobs, inMaxJobs);
+
+        OLO_CORE_INFO("JoltJobSystemAdapter initialized - MaxJobs: {0}, MaxBarriers: {1}", inMaxJobs, inMaxBarriers);
+    }
+
+    JoltJobSystemAdapter::~JoltJobSystemAdapter()
```

Clang (14.0.6)

[warning] use '= default' to define a trivial destructor

(modernize-use-equals-default)